pants2

```diff
@@ -4,11 +4,13 @@
 
 # This bootstrap script invokes Pants using a Python 2 interpreter.
 
+REPO_ROOT=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd -P)
+
 export PY="${PY:-python2.7}"
 
 # Allow spawned subprocesses, such as unit tests, to execute with either Python 2 and Python 3.
 # So long as the target does not have a compatibility constraint that requires only Python 3, the
 # interpreter selection will default to using Python 2 as this is the minimum acceptable interpreter.
 export PANTS_PYTHON_SETUP_INTERPRETER_CONSTRAINTS="${PANTS_PYTHON_SETUP_INTERPRETER_CONSTRAINTS:-['CPython>=2.7,<3','CPython>=3.6,<4']}"
 
-./pants "$@"
+${REPO_ROOT}/pants "$@"
```

Comment:
Should we be using `exec` here? This was my fault for not originally using it, as I didn't know what exec did.

I think it's fine as is, but more correct to add `exec`.